recipes/pyxtensor-feedstock/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "pyxtensor" %}
+{% set version = "0.0.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tdegeus/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 3b1c7f0f1d5013aff15f3a78361ca6d5a0f8f9ceacc258d703fa46fa801dcbdf
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
```

Probably needs compilers
```suggestion
  build:
   - {{ compiler('c') }}
```

---

I added a C++ compiler